apps/app/.env.staging

```diff
@@ -1,7 +1,5 @@
 NEXT_PUBLIC_NAME=STAGING
-NEXT_PUBLIC_API_BASE_URL=https://apiv4-staging.devlaunchers.org
-NEXT_PUBLIC_API_URL=https://apiv4-staging.devlaunchers.org/api
-NEXT_PUBLIC_STRAPI_URL=https://apiv4-staging.devlaunchers.org/api
-NEXT_PUBLIC_GOOGLE_AUTH_URL=https://apiv4-staging.devlaunchers.org/api/connect/google
-NEXT_PUBLIC_DISCORD_AUTH_URL=https://discord.com/api/oauth2/authorize?client_id=815294711983112194&redirect_uri=https%3A%2F%2Fapi-staging.devlaunchers.org%2Fusers%2Fauth%2Fdiscord%2Fcallback&response_type=code&scope=identify
-NEXT_PUBLIC_FRONT_END_URL=https://staging.devlaunchers.org
```

Comment:
is there any reason we are getting rid of the `NEXT_PUBLIC_FRONT_END_URL` variable?